datalad/support/gitrepo.py

```diff
@@ -1029,20 +1028,28 @@ def _git_custom_command(self, files, cmd_str,
             else cmd_str + files
         assert(cmd[0] == 'git')
         cmd = cmd[:1] + self._GIT_COMMON_OPTIONS + cmd[1:]
-        return self.cmd_call_wrapper.run(cmd, log_stderr=log_stderr,
-                                  log_stdout=log_stdout, log_online=log_online,
-                                  expect_stderr=expect_stderr, cwd=cwd,
-                                  env=env, shell=shell, expect_fail=expect_fail)
+        return self.cmd_call_wrapper.run(
+            cmd,
+            log_stderr=log_stderr,
+            log_stdout=log_stdout,
+            log_online=log_online,
+            expect_stderr=expect_stderr,
+            cwd=cwd,
+            env=env,
+            shell=shell,
+            expect_fail=expect_fail)
 
 # TODO: --------------------------------------------------------------------
 
-    def add_remote(self, name, url, options=[]):
+    def add_remote(self, name, url, options=None):
         """Register remote pointing to a url
         """
+        cmd = ['git', 'remote', 'add']
+        if options:
+            cmd += options
+        cmd += [name, url]
 
-        return self._git_custom_command(
-            '', ['git', 'remote', 'add'] + options + [name, url]
-        )
+        return self._git_custom_command('', cmd)
```

Comment:
spaghetti lovers! From one easy to digest construct to 5 lines. What for? It was beautifully pythonic and without any bug. Default [] could have no side effect since was added. I like pep8 but I am not religious